docs/prow/prow-jobs.md

```diff
@@ -0,0 +1,84 @@
+# Prow jobs
+
+This document provides an overview of Prow jobs.  
+
+## Directory structure
+
+Prow jobs reside in the `prow/jobs` directory in the `test-infra` repository. Job definitions are configured in `yaml` files, and can be connected to specific components or be more general, like for integration jobs. General jobs are defined directly under the `jobs/{repository_name}` directory. Jobs configured for components are available in `jobs/{repository_name}`directory which includes subdirectories representing each component and containing job definitions. 
+
+
+This is a sample directory structure:
+
+```
+...
+prow
+|- cluster
+| |- starter.yaml
+|- images
+|- jobs
+| |- kyma
+| | |- components
+| | | |- environments
+| | | | |- environments.yaml
+| | |- kyma.integration.yaml
+|- scripts
+|- config.yaml
+|- plugins.yaml
+...
+```
+> **NOTE:** All `yaml` files in the whole `jobs` structure need to have unique names.
+
+## Job types
+
+You can configure the following job types:
+
+- **Presubmit** jobs run on pull requests (PRs). They validate changes against the target repository. By default, all presubmit jobs are required to pass before you can merge the PR.  If you set the **optional** parameter to `true`, a job becomes optional and you can still merge your PR even if the job fails. 
+- **Postsubmit** jobs are almost the same as the already defined presubmit jobs, but they run after the PR is merged. You can notice the difference in labels, as the postsubmit job uses **preset-build-master** instead of **preset-build-pr**.
+- **Periodic** jobs run automatically at a scheduled time. You don't need to modify or merge the PR to trigger them. 
+
+The presubmit and postsubmit jobs for a PR run in random order, and their number for a PR depends on the configuration in the `yaml` file. You can check the job status on`https://status.build.kyma-project.io/`.
+
+
+## Naming convention 
+
+When you define jobs for Prow, the **name** parameter of the job must follow one of these patterns:
+
+- `{prefix}-{repository-name}-{component-name}-{job-name}` for components
```

Comment:
hm... I'm not sure if it is true. For example, we have a job: `pre-master-kyma-components-api-controller`, where:
 - prefix: `pre-master`
 - repository-name: `kyma`
 - path: `components`
 - component-name: `api-controller`

In general, our job names are like that:
`{prefix}-{repository}-{path-to-component}`

But If component will have more jobs then it will look like that:
`{prefix}-{repository}-{path-to-component}-{job-name}`

Please take a note that our components are not always in `components` directory, some examples:
```
pre-master-kyma-docs
pre-master-website-tools-documentation-generator
pre-master-kyma-tests-application-operator-tests
pre-master-website
pre-master-website-governance
```

So the main job for repository/component is without `-{job-name}`